azerothcore/azerothcore-wotlk
# fix(Core): fix ip bans

## Changes Proposed:
- Fix the table `ip_banned` not being taken into account when logging in.

## Issues Addressed:

None

## SOURCE:


## Tests Performed:

- Applied C++ fix
- Tested on Windows 10 server and client

## How to Test the Changes:


1. Run the command `ban ip 127.0.0.1 30s testing`.
2. Make sure you can't log in in the next 30 seconds.
3. Make sure you can log in again after the 30 seconds ban.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
